src/metabase/util.cljc

```diff
@@ -8,7 +8,7 @@
    [clojure.walk :as walk]
    [flatland.ordered.map :refer [ordered-map]]
    [medley.core :as m]
-   [metabase.shared.util.i18n :refer [tru] :as shared.i18n]
+   [metabase.shared.util.i18n :refer [tru] :as i18n]
```

Comment:
I was so confused how this changed an alias for a require and no call sites broke.

Reply:
yeah nothing was actually using `shared.i18n`. Not sure why Kondo didn't error about this being an unused `:require`